ephios/core/signals.py

```diff
@@ -69,4 +78,49 @@ def register_base_consequence_handlers(sender, **kwargs):
     dispatch_uid="ephios.core.signals.send_participation_state_changed_mail",
 )
 def send_participation_state_changed_mail(sender, instance, **kwargs):
-    mail.participation_state_changed(instance)
+    from ephios.core.models import AbstractParticipation
+
+    if instance.state == AbstractParticipation.States.CONFIRMED:
+        from ephios.core.notifications.types import ParticipationConfirmedNotification
+
+        ParticipationConfirmedNotification.send(instance)
```

Comment:
@felixrindt you'd prefer sending those from the disposition and not from the post_save, don't you?